elastica/surface/plane.py

```diff
@@ -0,0 +1,32 @@
+__doc__ = """plane surface class"""
+
+from elastica.surface.surface_base import SurfaceBase
+import numpy as np
+from numpy.testing import assert_allclose
+from elastica.utils import Tolerance
+
+
+class Plane(SurfaceBase):
+    def __init__(self, plane_origin, plane_normal):
+        """
+        Plane surface initializer.
+
+        Parameters
+        ----------
+        plane_origin: numpy.ndarray
+        2D (dim, 1) array containing data with 'float' type.
+        Origin of the plane.
+        plane_normal: numpy.ndarray
+        2D (dim, 1) array containing data with 'float' type.
+        The normal vector of the plane, must be normalized.
+        """
+
+        assert_allclose(
+            np.linalg.norm(plane_normal),
+            1,
+            atol=Tolerance.atol(),
+            err_msg="plane normal is not a unit vector",
+        )
```

Comment:
Can you add a test to check error message

Reply:
done